lighthouse-core/report/html/renderer/dom.js

```diff
@@ -213,12 +213,12 @@ class DOM {
   /**
    * Guaranteed context.querySelector. Always returns an element or throws if
    * nothing matches query.
-   * @param {string} query
+   * @template {string} T
+   * @param {T} query
    * @param {ParentNode} context
-   * @return {!HTMLElement}
    */
   find(query, context) {
-    /** @type {?HTMLElement} */
+    /** @type {?import('typed-query-selector/parser').ParseSelector<T>} */
```

Comment:
what are your thoughts about doing it this way vs using the `querySelector` shim provided by default?

Reply:
I think you're suggesting deleting this jsdoc line and relying on the types the lib injects. however, the `T` type doesn't transfer. quick to test: just delete the line and see how the function always returns `Element` then.